service/gradle/dependencies.gradle

```diff
@@ -22,10 +22,19 @@ dependencies {
   implementation 'com.azure:azure-storage-blob:12.17.1'
   implementation group: 'com.azure.resourcemanager', name: 'azure-resourcemanager-monitor', version: '2.18.0'
 
+  // AWS dependencies
+  implementation platform('software.amazon.awssdk:bom:2.20.22')
+  implementation 'software.amazon.awssdk:arns'
+  implementation 'software.amazon.awssdk:auth'
+  implementation 'software.amazon.awssdk:regions'
+  implementation 'software.amazon.awssdk:s3'
+  implementation 'software.amazon.awssdk:sts'
+
   // Terra deps
   implementation group: "bio.terra", name: "datarepo-client", version: "1.41.0-SNAPSHOT"
   implementation group: "bio.terra", name:"billing-profile-manager-client", version: "0.1.29-SNAPSHOT"
   implementation group: "bio.terra", name:"terra-policy-client", version:"0.2.38-SNAPSHOT"
+  implementation group: "bio.terra", name:"terra-aws-resource-discovery", version:"v0.4.3-SNAPSHOT"
```

Comment:
@dexamundsen is having to include the `v` in the version for `terra-aws-resource-discovery` dependencies expected with the bumper config for that repo?  Without the `v` the artifact cannot be resolved.